### PR TITLE
Add gethostlatency example

### PR DESCRIPTION
**Pull request by acj**
* what problem are you trying to solve? (or if there's no problem, what's the motivation for this
  change?)
  * Make it possible to use uprobes and uretprobes for the same symbol
* what changes does this pull request make?
  * Fix hash key collisions so that uprobes and uretprobes for the same symbol can be used in the same program. I followed the same `p_` and `r_` convention that kprobes use
  * Add example of using uprobes and uretprobes together (gethostlatency)
* are there any non-obvious implications of these changes? (does it break compatibility with previous
  versions, etc)
  * I don't think so

### CODE REVIEW

File: examples/gethostlatency.rs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            .binary(libc_so_path)
+            .symbol(symbol)
+            .pid(pid)
+            .attach(&mut module)?;
+    }
+
+    // the "events" table is where the "function was called" events get sent
+    let table = module.table("events")?;
+    // install a callback to print out latency events when they happen
+    let mut perf_map = init_perf_map(table, perf_data_callback)?;
```
Comment (javierhonduco) [inline examples/gethostlatency.rs:83]: `init_perf_map` [will be be deprecated in favour of `PerfMapBuilder::new(table, cb).build()`](https://github.com/rust-bpf/rust-bcc/blob/5053fc9a4863c0556a1959a52edb4a1da1dbb793/src/perf_event/perf_map.rs#L72)

**Review by javierhonduco — APPROVED** [review]
LGTM

File: examples/gethostlatency.rs (hunk 1 above)
Comment (brayniac) [reply examples/gethostlatency.rs:83]: Agree. I would definitely prefer the new style be used in examples in this repo. We will mark `init_perf_map()` as deprecated at some point. I just opened #142 to track that.

**Review by brayniac — CHANGES_REQUESTED** [review]
Thanks for the PR! I've left some feedback inline that I'd like addressed before merging. For the most part this looks good. Thanks for the fix and for adding an example to validate.

**Review by brayniac — APPROVED** [review]
Thanks for making the requested changes. This PR looks good now

**Pull request merged**